.github/workflows/repo-check.yml

```
@@ -19,8 +19,8 @@ on:
   pull_request:
 
 jobs:
-  copyright-check:
-    runs-on: ubuntu-18.04 # needed for astyle 2.05.1
+  repo-check:
```

Maybe "headers-check" is a better name? The workflow is already called repo-check.
```suggestion
  headers-check:
```

---

Except that it's checking every file not just headers, isn't it? I don't feel strongly about the name. We can certainly change it.

---

I see. I meant "file header", the beginning of a file. But there is certainly a confusion with C++ header files...

---

That makes sense. Would `file-headers-check` be too long?

---

No. If you think that’s enough to avoid the confusion, that works for me.
It seems that the generic term is "intellectual property", and `intellectual-property-check` seems too much, doesn’t it?

---

```suggestion
  file-headers-check:
```

---

I think `file-headers-check` will work well. Technically these checks are scanning the entire file looking for lines starting with comment strings so there's probably no concise way to state exactly what they are looking for. As you say `intellectual-property-check` may be the most accurate but it does seem a bit much.

---

Another name some people use is `banner`. But I think `file-headers-check` is probably sufficient :)